packages/@sanity/desk-tool/src/components/BrokenReferences.js

```diff
@@ -90,7 +105,13 @@ const BrokenReferences = streamingComponent(props$ =>
         filter(docs => docs.length === ids.length),
         map(docs => docs.filter(isMissingPublished)),
         map(broken =>
-          broken.length > 0 ? <BrokenRefs documents={broken}>eek</BrokenRefs> : props.children
+          broken.length > 0 ? (
+            <BrokenRefs documents={broken} type={type} schema={schema}>
+              eek
```

Comment:
Should remove the children here (`eek`) since it's not actually used. Just leftovers from my testing - sorry about that.